### ISSUE TITLE
Managing Signal to Noise Ratio - OCTO Ticket #2 of 6

### ISSUE THREAD

**Issue by jknipes** [department-of-veterans-affairs/va.gov-team#78823]
This is the second ticket generated from #76094. Once this section is ready, it will be moved to the Draft Space for editing. 

- [x] Doc now exists in the drafts space: https://vfs.atlassian.net/wiki/spaces/PWD/pages/3043164489/DRAFT+Managing+Signal+to+Noise+Ratio

### TASKS
- [x] Format page for deployment
- [x]  Edit Page for deployment
- [x]  Flag any issues that need POC/SME expertise to fix
- [x]  Make in-Line Comments for SME/POC

### DEFINITION OF DONE
- [x] Document has been reviewed, edited, formatted, in-line comments made, and any issues flagged for POC/SME

**Comment by jknipes** [comment]
@BSmolensky could you please assist in fleshing out the work and DOD for this ticket? Thanks!

**Comment by BSmolensky** [comment]
Do you want that work here in Github or in Zenhub, @jknipes ?

**Comment by BSmolensky** [comment]
@jknipes - Ticket fleshed out. FYI: ticket is blocked until doc is moved to drafts space.

**Comment by BSmolensky** [comment]
@acrollet - I was able to move the doc to the drafts space myself. So the doc now lives in the Platform Website Drafts fold. (You can find the link in the description above). As I was moving it, I noticed that the page you linked to has a child page. I moved both pages to the Draft folder. But, the child page is not part of the scope of this ticket. **Do you know what the plan for the child page is?** 

Looking forward to working with you on this. Have a great Friday and a wonderful weekend.

**Comment by BSmolensky** [comment]
@acrollet - After looking over this page and referencing my past conversations with Mike Chelen, I realize that Mike's original plan for the Monitor and Alerting Best Practices page was for it to get broken up into smaller, stand-alone pages. **Is that still the plan for this page?**

I took a look at your ticket related to this page (Guidance on improving signal to noise ratio on alerting #70830) and what I gather is that the next step is to break off the Managing Signal to Noise Ratio section of the mother page and make that a stand-alone page. 

I took a guess and figured that this is what you would like to happen. So, I have created a DRAFT of the stand alone document: [Managing Signal to Noise Ratio](https://vfs.atlassian.net/wiki/spaces/PWD/pages/3043164489/DRAFT+Managing+Signal+to+Noise+Ratio). I have edited this document, polished certain sections, formatted it for deployment, and checked it for any issues that I might need you to resolved. There are none. 

**Is this the page you would like deployed?** 
If it is, then it is good to go. All I would need from you is the location in the IA where you would like it to live. The previous doc that I worked on with Mike now lives in `Developer docs / Monitoring performance / Monitor Applications and Infrastructure with Datadog`. Let me know if you would like this new signal to noise ration doc to live there too. Once I know where you would like to to live, I will remove the word "Draft" from the title, move it to its final home in the IA, and our team will deploy it. 

If however this is NOT what the plan is now. If, in fact, you would like the entire Best Practices doc to get deployed, then please let me know and I will finished editing, formatting, polishing, and checking the rest of the document. 

Again, I hope I am right about what the path forward looks like. If I am off the mark, I am sorry. I can get back on the mark asap. 

Hope you had a great weekend.

**Comment by acrollet** [comment]
@BSmolensky thank you for your work on this! I've updated the draft doc with a few minor edits, could you check them and revert any you don't agree with? To answer your questions in order:

- yes, the best practices page should be split into stand-alone pages as you're doing
- yes, we would like this place to be deployed below `Developer docs / Monitoring performance / Monitor Applications and Infrastructure with Datadog` in whichever order you think appropriate.

**Comment by BSmolensky** [comment]
@acrollet - Thanks for the quick replay...and for working so early in your Mountain Time Zone this morning. I have no issues with your changes. Thanks for taking such a thorough look at the doc. I will put it in the IA and our team will deploy it later today. Thanks again for your help.

**Comment by BSmolensky** [comment]
Page is now in the IA and ready to deploy. This ticket is ready to close. 

Page Location: https://vfs.atlassian.net/wiki/spaces/pilot/pages/3043164489

**Comment by jknipes** [comment]
Work completed, moving to _Done_
